src/hooks/supabase-swr/hooks/useSelectSingle.ts

```diff
@@ -12,4 +13,17 @@ const useSelectSingle = <Data>(
     return useSWR(query, fetcher, swrConfig);
 };
 
+export const useSelectSingleNew = <Data>(
+    fetcher: any,
+    swrConfig?: Omit<SWRConfiguration, 'fetcher'>
+): SWRResponse<SuccessResponseSingle<Data>, PostgrestError> => {
+    const key = useMemo(() => (fetcher ? fetcher.url.href : null), [fetcher]);
+    const fetchFunction = useMemo(
+        () => () => fetcher.throwOnError().single(),
+        [fetcher]
+    );
+    console.log('select single new', key);
```

Comment:
Are you intentionally leaving this `console.log` behind?

Reply:
I was... but I feel like this is working well enough to not need anymore. I'll remove.